pkg/controllers/dashboard/systemcharts/controller_test.go

```diff
@@ -110,6 +114,47 @@ func Test_ChartInstallation(t *testing.T) {
 			name: "installation with image override",
 			setup: func(ctrl *gomock.Controller) chart.Manager {
 				settings.ConfigMapName.Set("fail")
+				settings.RancherWebhookVersion.Set("2.0.1")
+				manager := fake.NewMockManager(ctrl)
+				expectedValues := map[string]interface{}{
+					"capi": map[string]interface{}{
+						"enabled": features.EmbeddedClusterAPI.Enabled(),
+					},
+					"mcm": map[string]interface{}{
+						"enabled": features.MCM.Enabled(),
+					},
+					"global": map[string]interface{}{
+						"cattle": map[string]interface{}{
+							"systemDefaultRegistry": "",
+						},
+					},
+					"image": map[string]interface{}{
+						"repository": "rancher-test.io/rancher/rancher-webhook",
+					},
+				}
+				var b bool
+				manager.EXPECT().Ensure(
+					namespace.System,
+					"rancher-webhook",
+					"",
+					"2.0.1",
+					expectedValues,
+					gomock.AssignableToTypeOf(b),
+					"rancher-test.io/"+settings.ShellImage.Get(),
+				).Return(nil)
+
+				manager.EXPECT().Uninstall(operatorNamespace, "rancher-operator").Return(nil)
+				return manager
+			},
+			registryOverride: "rancher-test.io",
+		},
+		// TODO (maxsokolovsky) remove once the rancher-webhook-min-version setting is fully removed.
```

Comment:
We can remove this TODO and create a new issue to remove it from the code. similar to https://github.com/rancher/rancher/issues/38535

Reply:
Pasting the new issue here: https://github.com/rancher/rancher/issues/41561.